test/System.IdentityModel.Tokens.Jwt.Tests/JwtSecurityTokenHandlerTests.WithContextSwitches.cs

```diff
@@ -1,66 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using Microsoft.IdentityModel.JsonWebTokens;
-using Microsoft.IdentityModel.TestUtils;
-using Microsoft.IdentityModel.Tokens;
-using Xunit;
-
-namespace System.IdentityModel.Tokens.Jwt.Tests
-{
-    [CollectionDefinition("JwtSecurityTokenHandlerTestsWithContextSwitches", DisableParallelization = true)]
-    public class JwtSecurityTokenHandlerTestsWithContextSwitches
-    {
-        [Theory]
-        [InlineData(SecurityAlgorithms.RsaOAEP, true)]
-        [InlineData(SecurityAlgorithms.RsaOaepKeyWrap, false)]
-        public void JwtSecurityTokenHandler_CreateToken_AddShortFormMappingForRsaOAEP(string algorithm, bool useShortNameForRsaOaepKey)
-        {
-            AppContext.SetSwitch(X509EncryptingCredentials._useShortNameForRsaOaepKey, useShortNameForRsaOaepKey);
-            var encryptingCredentials = new X509EncryptingCredentials(Default.Certificate);
-            JwtSecurityTokenHandler tokenHandler = new JwtSecurityTokenHandler();
-
-            JwtSecurityToken token = CreateJwtSecurityToken(tokenHandler, encryptingCredentials);
-
-            Assert.Equal(token.Header.Alg, algorithm);
-
-            AppContext.SetSwitch(X509EncryptingCredentials._useShortNameForRsaOaepKey, false);
```

Comment:
we don't need the tests? I would think removing the AppContext.SetSwitch would be enough?

Reply:
What about the tests in X509EncryptingCredentialsTests.cs?

Reply:
I think it would make sense to change all the occurrences of `SecurityAlgorithms.RsaOaepKeyWrap` in tests to `RsaOAEP`, since the short name is default now.

Reply:
I agree with @kellyyangsong but can you also write a quick negative test to validate our default is now the short form. Thanks!

Reply:
+1 Add a test that asserts the behavior when the long form of the name is used.

Reply:
We have multiple test cases here: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/a2486af8f2eb5d6e470d0fb6fd1d3c1bf606e49a/test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandlerTests.cs#L2764

Reply:
As long as we have test coverage and not losing any, there is no concern.